test/t/unit/test_unit_count_args.py

```diff
@@ -18,7 +18,7 @@ def functions(self, bash):
     def _test(self, *args, **kwargs):
         return self._test_unit("_comp__test_unit %s", *args, **kwargs)
 
-    def test_1(self, bash):
+    def test_1(self, bash, functions):
```

Comment:
I don't think this is needed here - this doesn't rely on _comp__test_unit

Reply:
Thanks for catching this. I just pushed a revised version.